packages/compiler/core/parser.ts

```diff
@@ -123,7 +123,7 @@ interface UndecoratedListKind extends ListKind {
  * The fixed set of options for each of the kinds of delimited lists in Cadl.
  */
 // eslint-disable-next-line @typescript-eslint/no-namespace
-namespace ListKind {
+export namespace ListKind {
```

Comment:
This one surprises me, which public API exposes it? This was intended as a parser implementation detail.

Reply:
I can't find any references to ListKind other than the declaration in the .md.

Is it maybe just a weirdness with namespaces? I could refactor it to not be a namespace if so.